ime/app/src/main/java/com/anysoftkeyboard/keyboards/AnyKeyboard.java

```diff
@@ -674,13 +674,16 @@ protected boolean setupKeyAfterCreation(AnyKey key) {
             return true;
         }
 
-        // filling popup res for external keyboards
+        // if only popupCharacters are specified, call super nevertheless --
+        // we might want to add more characters.
+        /*
```

Comment:
I see that this logic has moved to a different class. Please remove this from here completely, we don't need it.